src/components/pages/Projects.css

```diff
@@ -0,0 +1,30 @@
+h5 {
+    text-align: left;
+    font-family: Bahnschrift;
+    font-size: 15px;
+    font-weight: bold;
+    padding: 0px 15px 15px;
+    margin-left: 90px;
+    margin-right: 90px;
+    display: inline-block;
+  }
+
+.info_wrapper > p {
+    text-align: left;
+    font-family: Bahnschrift;
+    font-size: 14px;
+    padding: 15px 15px;
+    font-weight: lighter;
+    margin-left: 90px;
+    margin-right: 90px;
+    line-height: 20px;
+    box-sizing:border-box;
+  }
```

Comment:
A side note on this (not super important but cleaner code): Notice that many of these properties are repeated. Let's see if we can combine them to make the code simpler and avoid repetition in the CSS attributes.